tests/posix/pthread_rwlock/src/posix_rwlock.c

```diff
@@ -80,7 +80,7 @@ static void test_rw_lock(void)
 
 	/* Creating N premptive threads in increasing order of priority */
 	for (i = 0; i < N_THR; i++) {
-		pthread_attr_destroy(&attr[i]);
+		memset(&attr[i], 0x0, sizeof(pthread_attr_t));
```

Comment:
No need for the memset, pthread_attr_init is the one meant to initialize the attr[i]

Reply:
this is just to make sure, if we are not initializing any parameters in the attr_init() this set the fields to 0x0.

Reply:
I understand that. I meant it is not, and should not be, necessary.

Reply:
Ok, will fix it.